plugins/modules/ome_alert_policies_message_id_info.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell OpenManage Ansible Modules
+# Version 8.2.0
+# Copyright (C) 2023 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: ome_alert_policies_message_id_info
+short_description: Get message ID information of alert policies.
+version_added: "8.2.0"
+description:
+   - "This module retrieves the message ID information of alert policies for OpenManage Enterprise
+      and OpenManage Enterprise Modular."
+extends_documentation_fragment:
+  - dellemc.openmanage.ome_auth_options
+requirements:
+    - "python >= 3.9.6"
+author: "Shivam Sharma (@ShivamSh3)"
+notes:
+    - Run this module from a system that has direct access to Dell OpenManage Enterprise
+      or OpenManage Enterprise Modular.
+    - This module supports C(check_mode).
+    - This module supports IPv4 and IPv6 addresses.
+'''
+
+EXAMPLES = r'''
+---
+- name: Get message ID details of all alert policies
+  dellemc.openmanage.ome_alert_policies_message_id_info:
+    hostname: "192.168.0.1"
+    username: "username"
+    password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+'''
+
+RETURN = r'''
+---
+msg:
+  description: Successfully retrieved alert policies message ids information.
```

Comment:
Description is not correct for msg. Please update.

Reply:
updated